src/App.tsx

```diff
@@ -50,15 +40,15 @@
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector value={author} />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && loaded && <Loader />}
```

Comment:
This condition seems incorrect. The Loader should be shown when the data is being loaded, which typically means `!loaded` rather than `loaded`. Please verify the logic here.